lisa/notifiers/text_result.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+from dataclasses import dataclass
+from typing import Any, List, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import notifier, schema
+from lisa.testsuite import TestResultMessage
+from lisa.util import constants, dump_file
+
+
+class TextResult(notifier.Notifier):
+    """
+    Creating log notifier to dump text formatted results for easier 
+    view in editing mode.
+    """
+    
+
+    @classmethod
+    def type_name(cls) -> str:
+        return constants.NOTIFIER_LOG_FILE
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return schema.Notifier
+
+    def _received_message(self, message: TestResultMessage) -> None:
+        if message.is_completed:
+            dump_file(
+                message.environment.log_path / "log_file.txt", 
```

Comment:
The log result should be in the run folder, not related to an environment. Refer to current root log names, this name should be like `lisa-20210818-004408-883-result.txt`. The path can be got like `"{constants.RUN_LOCAL_PATH}/lisa-{constants.RUN_ID}-result.txt")`